### PR TITLE
[11.0][PORT] pos_product_lot, product_lot

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

File: pos_product_lot/static/src/js/tour.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    }, {
+        trigger: ".o_pos_kanban button.oe_kanban_action_button",
+        content: "<p>Click to start the point of sale interface. It <b>runs on tablets</b>, laptops, or industrial hardware.</p><p>Once the session launched, the system continues to run without an internet connection.</p>",
+        position: "bottom",
+    }, {
+        trigger: '.o_main_content:has(.loader:hidden)',
+        content: 'waiting for loading to finish',
+        run: function () {
+            // it's a check
+        }
```
Comment (yelizariev) [inline pos_product_lot/static/src/js/tour.js]: Add more steps according to workflow

File: pos_product_available/static/src/js/tour.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    }, {
+        trigger: ".o_pos_kanban button.oe_kanban_action_button",
+        content: "<p>Click to start the point of sale interface. It <b>runs on tablets</b>, laptops, or industrial hardware.</p><p>Once the session launched, the system continues to run without an internet connection.</p>",
+        position: "bottom",
+    }, {
+        trigger: '.o_main_content:has(.loader:hidden)',
+        content: 'waiting for loading to finish',
+        run: function () {
+            // it's a check
+        }
```
Comment (yelizariev) [inline pos_product_available/static/src/js/tour.js]: Add more tests: 
* Buy products
* Check that quantity is decreased

**Review by yelizariev — CHANGES_REQUESTED** [review]
Add more tests or create an issue with the scenarios (one issue per each module)

**Comment by yelizariev** [comment]
@Railia check that new tours are enough to test workflow

File: pos_product_lot/static/src/js/tour.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    });
+
+    steps = steps.concat(goto_payment_screen_and_select_payment_method());
+
+    steps = steps.concat(generate_payment_screen_keypad_steps("5.94"));
+
+    steps = steps.concat(finish_order());
+
+    steps = steps.concat({
+        trigger: '.product .not-available:contains("-1")',
```
Comment (yelizariev) [inline pos_product_lot/static/src/js/tour.js:112]: Test that amount of unpacked products were increased

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: pos_product_lot/static/src/js/tour.js (hunk 3 above)
Comment (Railia) [reply pos_product_lot/static/src/js/tour.js:112]: @GabbasovDinar Amount of unpacked products weren't increased. Quantity on hand shows negative value (should be positive). Forecasted value isn't equal On hand value.

**Comment by yelizariev** [comment]
Too much conflicts

**Comment by GabbasovDinar** [comment]
need a new PR with fixes

**Pull request closed without merging**